src/main/resources/assets/tinkersurvival/lang/zh_cn.json

```
@@ -11,107 +11,107 @@
+    "item.tinkersurvival.saw.description": "用来锯木头！",
+    "item.tinkersurvival.saw_blade_cast": "锯刃金质铸模",
     "item.tinkersurvival.saw_blade_sand_cast": "锯刃沙子铸模",
     "item.tinkersurvival.saw_blade_red_sand_cast": "锯刃红沙铸模",
     "item.tinkersurvival.reinforced_wool_helmet": "加固的羊毛头盔",
     "item.tinkersurvival.reinforced_wool_chestplate": "加固的羊毛胸甲",
     "item.tinkersurvival.reinforced_wool_leggings": "加固的羊毛护腿",
     "item.tinkersurvival.reinforced_wool_boots": "加固的羊毛靴子",
-    "item.tinkersurvival.reinforced_jelled_slime_helmet": "加固的史莱姆头盔",
-    "item.tinkersurvival.reinforced_jelled_slime_chestplate": "加固的史莱姆胸甲",
-    "item.tinkersurvival.reinforced_jelled_slime_leggings": "加固的史莱姆护腿",
-    "item.tinkersurvival.reinforced_jelled_slime_boots": "加固的史莱姆靴子",
+    "item.tinkersurvival.reinforced_jelled_slime_helmet": "加固的黏液头盔",
+    "item.tinkersurvival.reinforced_jelled_slime_chestplate": "加固的黏液胸甲",
+    "item.tinkersurvival.reinforced_jelled_slime_leggings": "加固的黏液护腿",
+    "item.tinkersurvival.reinforced_jelled_slime_boots": "加固的黏液靴子",
     "item.tinkersurvival.tinkers_survival": "匠魂生存手册",
     "item.tinkersurvival.tinkers_survival_modpack": "匠魂生存整合包手册",
     "item.tinkersurvival.bandage": "绷带",
-    "item.tinkersurvival.cloth": "布片",
+    "item.tinkersurvival.cloth": "布条",
     "item.tinkersurvival.crude_bandage": "粗制绷带",
     "item.tinkersurvival.ointment": "软膏",
     "item.tinkersurvival.plant_paste": "植物浆",
     "item.tinkersurvival.wooden_cup": "神秘的木杯子",
-    "block.tinkersurvival.rock_stone_block": "小块石头",
-    "block.tinkersurvival.stone_loose_rock": "松散的石子堆 (石头)",
-    "block.tinkersurvival.andesite_loose_rock": "松散的石子堆 (安山岩)",
-    "block.tinkersurvival.diorite_loose_rock": "松散的石子堆 (闪长岩)",
-    "block.tinkersurvival.granite_loose_rock": "松散的石子堆 (花岗岩)",
-    "block.tinkersurvival.sandstone_loose_rock": "松散的石子堆 (砂岩)",
-    "block.tinkersurvival.red_sandstone_loose_rock": "松散的石子堆 (红砂岩)",
-    "jei.tinkersurvival.description.rock": "可以在地上找到这些石块，与小刀合成可以获得燧石碎片。",
-    "jei.tinkersurvival.description.plant_fiber": "植物纤维会在用小刀破坏草时掉落。它们可以制作用于工具的植物绳和用于绷带药膏的植物浆",
-    "jei.tinkersurvival.description.stick": "木棍可以在破坏树叶时掉落，用小刀可以增加获得木棍的几率。",
-    "jei.tinkersurvival.description.flint_shard": "燧石碎片可以通过在任意石质方块表面敲击燧石获得。用燧石击打任意石质方块，它也许会碎成两块燧石碎片。",
+    "block.tinkersurvival.rock_stone_block": "小块岩石",
+    "block.tinkersurvival.stone_loose_rock": "松散的岩石堆（石头）",
+    "block.tinkersurvival.andesite_loose_rock": "松散的岩石堆（安山岩）",
+    "block.tinkersurvival.diorite_loose_rock": "松散的岩石堆（闪长岩）",
+    "block.tinkersurvival.granite_loose_rock": "松散的岩石堆（花岗岩）",
+    "block.tinkersurvival.sandstone_loose_rock": "松散的岩石堆（砂岩）",
+    "block.tinkersurvival.red_sandstone_loose_rock": "松散的岩石堆（红砂岩）",
+    "jei.tinkersurvival.description.rock": "你可以在地上捡到这些岩石，用燧石小刀可以将它切割成燧石碎片",
+    "jei.tinkersurvival.description.plant_fiber": "用小刀斩断草时会掉落植物纤维。植物纤维可以合成用于制作工具的草绳和制作绷带软膏的植物浆。",
+    "jei.tinkersurvival.description.stick": "破坏树叶时会掉落木棍。使用任何小刀都可以增加木棍掉落的概率。",
```

```suggestion
    "jei.tinkersurvival.description.stick": "破坏树叶时会掉落木棍。使用任何小刀都可以使木棍掉落的概率翻倍。",
```
original text: `Sticks are dropped from leaves. Using any knife doubles the chance of getting a stick.`
fixes the multiplier part

---

it's better, thanks